include/mbedtls/ssl.h

```diff
@@ -2243,15 +2243,6 @@ void mbedtls_ssl_conf_renegotiation_period( mbedtls_ssl_config *conf,
  *
  * \return         0 if nothing's pending, 1 otherwise.
```

Comment:
I suggest to make this “a positive value otherwise”, in case we want to encode what is pending in the return value in the future.